core/src/tests/test-parachain/src/lib.rs

```diff
@@ -115,14 +117,14 @@ parameter_types! {
 			weights.base_extrinsic = ExtrinsicBaseWeight::get();
 		})
 		.for_class(DispatchClass::Normal, |weights| {
-			weights.max_total = Some(NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT);
+			  weights.max_total = Some(( NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT ).set_proof_size(u64::MAX).into());
 		})
 		.for_class(DispatchClass::Operational, |weights| {
-			weights.max_total = Some(MAXIMUM_BLOCK_WEIGHT);
+			  weights.max_total = Some(MAXIMUM_BLOCK_WEIGHT.set_proof_size(u64::MAX).into());
 			// Operational transactions have some extra reserved space, so that they
 			// are included even if block reached `MAXIMUM_BLOCK_WEIGHT`.
 			weights.reserved = Some(
-				MAXIMUM_BLOCK_WEIGHT - NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT
+				  (MAXIMUM_BLOCK_WEIGHT - NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT).set_proof_size(u64::MAX).into()
```

Comment:
Can you add those to the centrifuge-chain repo update also?

Reply:
Yeah, that was on my plan for the cent chain `0.9.32` upgrade :)